src/persistentReducedStream.ts

```diff
@@ -39,13 +43,14 @@ export const persistentReducedStream = <State>(
   name: string,
   initialState: State,
   reducers: RegisteredReducer<State, any>[],
-  errorSubject: Subject<any> = defaultErrorSubject
+  { errorSubject, namespace }: PersistentReducedStreamOptions = {}
```

Comment:
It's a bit unfortunate that we have to introduce an option object here.
In a while, I think we should reconsider how much we have used `errorSubject` and consider converting back to normal arguments with `errorSubject` as the last, optional, one.

Reply:
Could you not use two signatures, one with the errorSubject and one without? rxjs does that a lot?

Reply:
I don't think that's worth it at this point. I'd like to delay introducing complexity like that as far as possible.
I expect that in a few months, the signature will end up as `..., namespace?: string, errorSubject?: Subject`, as we don't really use the `errorSubject` at all, it's just there for completeness sake